packages/mapviewer/src/modules/map/components/cesium/CesiumInteractions.vue

```diff
@@ -233,6 +243,62 @@ function onClick(event) {
     viewer.scene.requestRender()
 }
 
+function create3dKmlFeature(viewer, kmlFeature, kmlLayer) {
+    let geometry
+    let featureCoordinates
+    if (kmlFeature.id.position) {
+        const position = kmlFeature.id.position.getValue(viewer.clock.currentTime)
+        const cartographic = Cartographic.fromCartesian(position)
+
+        featureCoordinates = proj4(WGS84.epsg, WEBMERCATOR.epsg, [
+            // pos is in EPSG:4978 but proj4 does not support it so it is transformed to EPSG:4326 first
```

Comment:
little confusion here, `EPSG:4978` is pretty much equal to `EPSG:4326` (both are WGS84), here you are transforming into `EPSG:3857` (metric mercator)